cookbooks/aws-parallelcluster-entrypoints/recipes/finalize.rb

```diff
@@ -16,6 +16,11 @@
 
 fetch_config 'Fetch and load cluster configs'
 
+if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
```

Comment:
Since this is used in different part of the code, can we encapsulate this check in a helper function `custom_node_package?` this would make the code more readable and the recipes more maintainable.

Reply:
Addressing it.